packages/picasso.js/src/core/layout/dock/docker.js

```diff
@@ -415,7 +406,7 @@ function dockLayout(initialSettings) {
       throw new Error('Invalid rect');
     }
     if (!components.length) {
-      return { visible: [], hidden: [] };
+      return { visible: [], hidden: [], order: [] };
```

Comment:
`order` should be `ordered` here, right?

Reply:
yes